Lib/test/test_pty.py

```diff
@@ -131,41 +111,40 @@ def test_openpty(self):
         try:
             mode = tty.tcgetattr(pty.STDIN_FILENO)
         except tty.error:
-            # not a tty or bad/closed fd
+            # Not a tty or bad/closed fd.
             debug("tty.tcgetattr(pty.STDIN_FILENO) failed")
             mode = None
 
-        new_stdin_winsz = None
-        if self.stdin_rows is not None and self.stdin_cols is not None:
+        new_dim = None
+        if self.stdin_dim != None:
```

Comment:
just write this as `if self.stdin_dim:` there i no need to check specifically for None.

Reply:
Makes sense.